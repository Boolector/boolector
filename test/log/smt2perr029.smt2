(declare-fun fun (
  (_ BitVec 32))		; no supported for UF yet
