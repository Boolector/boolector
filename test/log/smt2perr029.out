log/smt2perr029.smt2:3:1: reached end-of-file but expected '(' or 'Bool'
